binary_search_tree/binary_search_tree.py

```diff
@@ -53,4 +55,9 @@ def contains(self, target):
         # return False
 
     def get_max(self):
-        pass
+        max = self.value
```

Comment:
you can also just check for the right-most node. since you know that any item to the right of a node must be larger than it, the fact that it exists means it's larger than it's parent - no need to keep a curr_max.